### PR TITLE
General code update

### PR DESCRIPTION
**Pull request by seb-exiin**
Modify some typos in descriptions and some spaces here and there

### CODE REVIEW

File: Source/UINavigation/Public/UINavWidget.h [hunk 1]
```diff
@@ -712,25 +713,28 @@ class UINAVIGATION_API UUINavWidget : public UUserWidget
...
 	class UUINavButton* FetchButtonByDirection(const ENavigationDirection Direction, UUINavButton* Button);
 
 	/**
-	*	Adds given widget to screen (strongly recomended over manual alternative)
+	*	Adds given widget to screen (strongly recommended over manual alternative)
 	*
 	*	@param	NewWidgetClass  The class of the widget to add to the screen
 	*	@param	bRemoveParent  Whether to remove the parent widget (this widget) from the viewport
 	*	@param  bDestroyParent  Whether to destruct the parent widget (this widget)
+	*	@param  ZOrder order to display the widget
```
Comment (goncasmage1) [inline Source/UINavigation/Public/UINavWidget.h:726]: Don't forget to start with upper case

File: Source/UINavigation/Private/UINavWidget.cpp [hunk 2]
```diff
@@ -2452,9 +2452,9 @@ void UUINavWidget::DispatchNavigation(const int Index, const bool bBypassForcedN
 
 		UUINavComponent* FromComponent = GetUINavComponentAtIndex(ButtonIndex);
 		UUINavComponent* ToComponent = GetUINavComponentAtIndex(Index);
+
```
Comment (goncasmage1) [inline Source/UINavigation/Private/UINavWidget.cpp:2455]: I'd rather keep this as-is.

File: Source/UINavigation/Private/UINavWidget.cpp [hunk 3]
```diff
@@ -2452,9 +2452,9 @@ void UUINavWidget::DispatchNavigation(const int Index, const bool bBypassForcedN
 
 		UUINavComponent* FromComponent = GetUINavComponentAtIndex(ButtonIndex);
 		UUINavComponent* ToComponent = GetUINavComponentAtIndex(Index);
+		
 		if (FromComponent != nullptr) FromComponent->OnNavigatedFrom();
 		if (ToComponent != nullptr) ToComponent->OnNavigatedTo();
-
```
Comment (goncasmage1) [inline Source/UINavigation/Private/UINavWidget.cpp:2457]: I'd rather keep this as-is.

File: Source/UINavigation/Private/UINavPCComponent.cpp [hunk 4]
```diff
@@ -95,7 +95,6 @@ void UUINavPCComponent::TickComponent(float DeltaTime, ELevelTick TickType, FAct
 			break;
 	}
```
Comment (goncasmage1) [inline Source/UINavigation/Private/UINavPCComponent.cpp:97]: Maybe remove this line as well?

File: Source/UINavigation/Private/UINavOptionBox.cpp [hunk 5]
```diff
@@ -12,6 +12,7 @@ void UUINavOptionBox::NativeConstruct()
 
 	if (!LeftButton->OnClicked.IsBound())
 		LeftButton->OnClicked.AddDynamic(this, &UUINavOptionBox::NavigateLeft);
+
```
Comment (goncasmage1) [inline Source/UINavigation/Private/UINavOptionBox.cpp:15]: I'd rather keep this as-is.

File: Source/UINavigation/Private/UINavComponentBox.cpp [hunk 6]
```diff
@@ -11,6 +11,7 @@ void UUINavComponentBox::NativeConstruct()
 
 	if (!LeftButton->OnClicked.IsBound())
 		LeftButton->OnClicked.AddDynamic(this, &UUINavComponentBox::NavigateLeft);
+
```
Comment (goncasmage1) [inline Source/UINavigation/Private/UINavComponentBox.cpp:14]: I'd rather keep this as-is.

File: Source/UINavigation/Public/UINavWidget.h [hunk 7]
```diff
@@ -702,6 +702,7 @@ class UINAVIGATION_API UUINavWidget : public UUserWidget
 	/**
 	*	Returns the button that will be navigated to according to the given direction, starting at the given button
 	*
+	*	@param Button  Required button type for the UI Navigation plugin
```
Comment (goncasmage1) [inline Source/UINavigation/Public/UINavWidget.h:705]: Simply describe as "Target UINavButton"

File: Source/UINavigation/Public/UINavWidget.h [hunk 8]
```diff
@@ -712,25 +713,28 @@ class UINAVIGATION_API UUINavWidget : public UUserWidget
 	*	Returns the next button to navigate to
 	*
 	*	@param	Direction  Direction of navigation
+	*	@param  Button  Required button type for the UI Navigation plugin
```
Comment (goncasmage1) [inline Source/UINavigation/Public/UINavWidget.h:716]: Simply describe as "Target UINavButton"

File: Source/UINavigation/Public/UINavWidget.h [hunk 9]
```diff
@@ -712,25 +713,28 @@ class UINAVIGATION_API UUINavWidget : public UUserWidget
...
 	UUINavWidget* GoToWidget(TSubclassOf<UUINavWidget> NewWidgetClass, const bool bRemoveParent, const bool bDestroyParent = false, const int ZOrder = 0);
 
 	/**
-	*	Adds given widget to screen (strongly recomended over manual alternative)
+	*	Adds given widget to screen (strongly recommended over manual alternative)
 	*
 	*	@param	NewWidget  Object instance of the UINavWidget to add to the screen
 	*	@param	bRemoveParent  Whether to remove the parent widget (this widget) from the viewport
 	*	@param  bDestroyParent  Whether to destruct the parent widget (this widget)
+	*	@param  ZOrder order to display the widget
```
Comment (goncasmage1) [inline Source/UINavigation/Public/UINavWidget.h:737]: Don't forget to start with upper case

**Review by goncasmage1 — CHANGES_REQUESTED** [review]

File: Source/UINavigation/Public/UINavWidget.h (hunk 1 above)
Comment (seb-exiin) [reply Source/UINavigation/Public/UINavWidget.h:726]: done

File: Source/UINavigation/Private/UINavPCComponent.cpp (hunk 4 above)
Comment (seb-exiin) [reply Source/UINavigation/Private/UINavPCComponent.cpp:97]: done

File: Source/UINavigation/Private/UINavComponentBox.cpp (hunk 6 above)
Comment (seb-exiin) [reply Source/UINavigation/Private/UINavComponentBox.cpp:14]: done

File: Source/UINavigation/Public/UINavWidget.h (hunk 7 above)
Comment (seb-exiin) [reply Source/UINavigation/Public/UINavWidget.h:705]: done

File: Source/UINavigation/Public/UINavWidget.h (hunk 8 above)
Comment (seb-exiin) [reply Source/UINavigation/Public/UINavWidget.h:716]: done

File: Source/UINavigation/Public/UINavWidget.h (hunk 9 above)
Comment (seb-exiin) [reply Source/UINavigation/Public/UINavWidget.h:737]: done

**Comment by seb-exiin** [comment]
changed applied, ready to re-review

**Review by goncasmage1 — APPROVED** [review]

**Pull request merged**